components/backends/sglang/src/dynamo/sglang/args.py

```
@@ -128,6 +128,13 @@ def parse_args(args: list[str]) -> Config:
 
     server_args = ServerArgs.from_cli_args(parsed_args)
 
+    if server_args.skip_tokenizer_init == False:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not server_args.skip_tokenizer_init:` for false checks

Replace with `not server_args.skip_tokenizer_init`

(E712)